avocado_vt/plugins/vt_joblock.py

```diff
@@ -0,0 +1,113 @@
+import errno
+import logging
+import os
+import re
+import random
+import string
+import sys
+
+from avocado.core import exit_codes
+from avocado.core.settings import settings
+from avocado.plugins.base import JobPre, JobPost
+
+from ..test import VirtTest
+
+
+class VTJobLock(JobPre, JobPost):
+
+    name = 'vt-joblock'
+    description = 'Avocado-VT Job Lock/Unlock'
+
+    def __init__(self):
+        self.log = logging.getLogger("avocado.app")
+        self.lock_dir = settings.get_value(section="plugins.vtjoblock",
```

Comment:
@clebergnu I mentioned it'd be nice to use `os.path.expanduser` here as the idea of using `~/...` in config is really teasing.